src/index.html

```diff
@@ -1,14 +1,142 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Human rights</title>
   <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-  <h1>Human rights</h1>
+  <h1 class="heading">Universal Declaration of Human Rights</h1>
+  <h2 class="subheading">Preamble </h2>
+  <p>Whereas recognition of the inherent dignity and of the equal and inalienable
```

Comment:
when you write some text inside tags move it to the new line and add proper indentations so that the code looks clear